packages/wells/src/client/api/wells.ts

```diff
@@ -236,4 +250,71 @@ export class Wells extends AssetsAPI {
 
     return await this.getByIds(assetIds);
   };
+
+  public filterPolygonSearch = async ({
```

Comment:
can we hide what is happening behind the sdk a bit more, so instead of `filterPolygonSearch` i would do just a `list` and internally the SDK would take care of doing my filters in whatever way is reasonable.

eg here: https://github.com/cognitedata/cognite-sdk-js/blob/master/samples/typescript/quickstart.ts we only `list`, we don't have a `filterByRoots`

Reply:
Precisely what II was about to say! So, the idea would be something like wells.list(polygon=[polygon string], blocks=[a list of block names], field=[a list of field names], ...)

Reply:
So, I guess the method already works as expected, it maybe just a matter of renaming to list and creating a "Geometry" class to input to the polygon filter, where you would have polygon string + crs input + crs output, with both crses defaulting to 4326?

Reply:
So we can't call the method list as the AssetAPI class we are extending already has a method called list. `listWells` would be choice, but that is a bit of duplication of wells when doing `wells.listWells`. And suggestions?